src/main/java/com/google/devtools/build/lib/analysis/BashCommandConstructor.java

```diff
@@ -0,0 +1,52 @@
+// Copyright 2019 The Bazel Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.devtools.build.lib.analysis;
+
+import com.google.common.collect.ImmutableList;
+import com.google.devtools.build.lib.actions.Artifact;
+import com.google.devtools.build.lib.analysis.actions.FileWriteAction;
+import com.google.devtools.build.lib.vfs.PathFragment;
+
+/**
+ * The class for constructing command line for Bash.
+ */
+public class BashCommandConstructor implements CommandConstructor {
+
+  private final PathFragment shellPath;
+  private final String scriptPostFix;
```

Comment:
How about `scriptNameTail` or `scriptNameSuffix`?
First I thought this stored script code (to be appended to the command script) instead of the end of the file's name.

Reply:
Oh, I was using the original name, but `scriptNameSuffix` does sound better.